src/GitHub.Api/Platform/DefaultEnvironment.cs

```diff
@@ -131,6 +132,23 @@ public NPath GitExecutablePath
             }
         }
 
+        private NPath nodeJsExecutablePath;
+
+        public NPath NodeJsExecutablePath
+        {
+            get
+            {
+                if (!nodeJsExecutablePath.IsInitialized)
+                {
+                    nodeJsExecutablePath = IsWindows
+                        ? UnityApplication.Parent.Combine("Data", "Tools", "nodejs", "node.exe")
+                        : UnityApplication.Combine("Contents", "Tools", "nodejs", "node");
```

Comment:
This should probably be using the value returned by `UnityEditor.EditorApplication. applicationContentsPath` (https://docs.unity3d.com/ScriptReference/EditorApplication-applicationContentsPath.html). We should probably add a `UnityApplicationContents` property to the environment so we can use it here. I'm fine with doing that on a separate PR, this isn't a blocker

Reply:
Tracked in #617